scalameter-core/src/main/scala/org/scalameter/execution/invocation/instrumentation/MethodInvocationCounterVisitor.scala

```diff
@@ -0,0 +1,45 @@
+package org.scalameter.execution.invocation.instrumentation
+
+import org.objectweb.asm.{Type, ClassVisitor, MethodVisitor, Opcodes}
+import org.scalameter.execution.invocation.InvocationCountMatcher
+import scala.collection.mutable
+
+
+/** Walks over methods in a class and instruments ones that match [[org.scalameter.execution.invocation.InvocationCountMatcher]].
+ *
+ * @param cv [[org.objectweb.asm.ClassVisitor]] that actually visits classes and methods
+ * @param pattern selects methods that needs to be instrumented
+ * @param counterClass class of counting class
+ * @param counterMethod method name of counting class
+ * @param currentIndex counting table start index
```

Comment:
Describe that it is actually a `var` that can be read after the visitor completes its traversal.